app/routes/index.js

```diff
@@ -6,21 +6,24 @@ function isChecked (data, option) {
   return !!data && data.includes(option)
 }
 
-function setLabelData (data, labelData) {
-  return labelData.map((label) => {
-    if (typeof (label.value) === 'string' && label.value !== 'divider') {
+function setAnswerOptions (data, answers) {
+  return answers.map((label) => {
+    const { value, hint, text } = label
+
```

Comment:
This is great , Thank you. could you just change the label name to answer, please :)  instead of label we can use answer , it is confusing we are looping through answers and write label